### DISCUSSION TITLE
Setting argument options

### DISCUSSION THREAD

**Discussion by paambaati** [jurassiscripts/velociraptor#69] [Q&A] [answered]
[Deno 1.12](https://deno.com/blog/v1.12) just landed yesterday, and it introduces a new `--shuffle=N` flag for the `deno test` command.

I'm using a simple script to first generate the seed number as part of a GitHub Action step, and I'd like to pass it to a `vr` script that will in turn pass it on to the `--shuffle` flag as its value. What's the best way to do this?

**Example**
I noticed that the `vr` help prints this – "Any additional arguments will be passed to the script."

So I tried this –

```bash
 $ VR_LOG=DEBUG vr test --shuffle=1234
Running > deno test --lock=lock.json --cached-only --shuffle 1234
```

I see this strips away the `=` and parses the flag and the value as 2 separate arguments. I'd expect `=` would be preserved, especially when there's no spaces surrounding it.

```diff
+ Running > deno test --lock=lock.json --cached-only --shuffle=1234
- Running > deno test --lock=lock.json --cached-only --shuffle 1234
```

**Comment by umbopepato** [comment, marked as answer]
Hey @paambaati, thanks for reporting! Can you confirm that

```shell
$ vr run test --shuffle=1234
     ^^^
```

yields the desired command?

Looks like the method we use to forward additional arguments in the `vr` command normalizes the options, see #70.

**Reply by paambaati** [reply]
@umbopepato Yep, including `run` in the command fixes it.
